docs/release-notes/8.1.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[release-notes-header-8.1.0]]
+== 8.1
+
+[discrete]
+[[release-notes-8.1.0]]
+=== 8.1.0
+
+[discrete]
+[[breaking-changes-8.1.0]]
+==== Breaking changes
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are re-used in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: https://github.com/elastic/kibana/pull/
+There are no breaking changes in 8.1.0.
+// end::breaking-changes[]
+
+[discrete]
+[[features-8.1.0]]
+==== Features
+* Improves copy for the privilege check on the Endpoint page ({pull}124118[#124118]).
+* Introduces a new *Host risk classification* column in the All hosts table on the *Hosts* page. In addition, a new *Host by risk* tab has been added to the *Hosts* page and host detail pages. From the *Host by risk* tab, you can access an explanation of how a host’s risk is calculated and scored ({pull}122980[#122980], {pull}122586[#122586], {pull}122018[#122018], {pull}121075[#121075], {pull}120487[#120487], {pull}119734[#119734]).
+* Introduces the ability to  bulk edit rule index patterns and tags ({pull}122635[#122635]).
+* Expands Endpoint per-policy artifact assignment to include endpoint event filters and host isolation IP exceptions ({pull}121879[#121879], {pull}121632[#121632]).
+* Adds the rule execution UUID field to alerts. In addition, the `kibana.alert.rule.execution.uuid` is now part of the alert data schema and can be found in the field browser for the Alert's table.({pull}113058[#113058]).
```

Comment:
We should probably include the actual rule UUID field here and wrap it in `code text`. 
```suggestion
* Adds the rule execution UUID field to alerts. In addition, the `kibana.alert.rule.execution.uuid` field is now part of the alert data schema and can be found in the field browser in the Alerts table.({pull}113058[#113058]).
```

Reply:
Resolving because added the recommended edits.